app/src/main/res/values/strings.xml

```diff
@@ -956,4 +956,6 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_shop_gone_replaced_answer_hint">Type kind of place here</string>
     <string name="quest_shop_gone_replaced_answer_error">"Don't know what this is. Consider leaving a note."</string>
     <string name="quest_shop_vacant_type_title">This shop has been vacant. What\'s here now?</string>
+    <string name="quest_construction_completes_at_known_date">Completes at known date</string>
```

Comment:
```suggestion
    <string name="quest_construction_completes_at_known_date">Finished at date...</string>
```
Even if you find your wording better, the "..." is important.

Reply:
I will go with yet another wording - "Opens at date…" (tripple dot character + make clear that it is when road be open, not when construction is finished - what matters in some cases)